issue.go

```diff
@@ -822,6 +834,41 @@ func (s *IssueService) Create(issue *Issue) (*Issue, *Response, error) {
 	return s.CreateWithContext(context.Background(), issue)
 }
 
+// CreateInBulkWithContext creates multiple issues or sub-tasks with a single API call from a JSON representation.
+// Jira API docs: https://docs.atlassian.com/jira/REST/latest/#api/2/issue-createIssues
+func (s *IssueService) CreateInBulkWithContext(ctx context.Context, issues []*Issue) ([]*Issue, *Response, error) {
+	apiEndpoint := "rest/api/2/issue/bulk"
+	req, err := s.client.NewRequestWithContext(ctx, "POST", apiEndpoint, IssueBulkCreate{Issues: issues})
+	if err != nil {
+		return nil, nil, err
+	}
+	resp, err := s.client.Do(req, nil)
+	if err != nil {
+		// incase of error return the resp for further inspection
+		return nil, resp, err
+	}
+
+	responseIssues := new(IssueBulkCreateResponse)
+	defer resp.Body.Close()
+	data, err := ioutil.ReadAll(resp.Body)
```

Comment:
This should probably be updated for Go 1.16 to use `io.ReadAll` @andygrunwald thoughts? I think this would require updating the module version to 1.16 though and could cause issues with users.